README.md

```diff
@@ -8,65 +8,244 @@ Warns and then closes issues and PRs that have had no activity for a specified a
+- [stale-issue-label](#stale-issue-label)
+- [stale-pr-label](#stale-pr-label)
+
+Default value: `7`
+
+#### days-before-issue-close
+
+Override [days-before-close](#days-before-close) but only for the idle number of days before closing the stale issues.
+
+Default value: unset
+
+#### days-before-pr-close
+
+Override [days-before-close](#days-before-close) but only for the idle number of days before closing the stale pull requests.
+
+Default value: unset
+
+#### stale-issue-message
+
+The message that will be added as a comment to the issues when the stale workflow mark it automatically as stale with a label.
+
+You can omit the comment by setting [skip-stale-issue-message](#skip-stale-issue-message) to `true`.
+
+Default value: unset
+
+#### stale-pr-message
+
+The message that will be added as a comment to the pull requests when the stale workflow mark it automatically as stale with a label.
+
+You can omit the comment by setting [skip-stale-pr-message](#skip-stale-pr-message) to `true`.
+
+Default value: unset
+
+#### close-issue-message
+
+The message that will be added as a comment to the issues when the stale workflow close it automatically after being stale for too long.
+
+Default value: unset
+
+#### close-pr-message
+
+The message that will be added as a comment to the pull requests when the stale workflow close it automatically after being stale for too long.
+
+Default value: unset
+
+#### stale-issue-label
+
+The label that will be added to the issues when automatically marked as stale.  
+It should also be the one you should use if you wish to manually mark as stale the issues.
+
+Default value: `Stale`
+
+#### close-issue-label
+
+The label that will be added to the issues when closed automatically.  
+It will be automatically removed if the issues are no longer closed nor locked.
+
+Default value: unset
+
+#### stale-pr-label
+
+The label that will be added to the pull requests when automatically marked as stale.  
+It should also be the one you should use if you wish to manually mark as stale the pull requests.
+
+Default value: `Stale`
+
+#### close-pr-label
+
+The label that will be added to the pull requests when closed automatically.  
+It will be automatically removed if the pull requests are no longer closed nor locked.
+
+Default value: unset
+
+#### exempt-issue-labels
+
+The label(s) that can exempt to automatically mark as stale the issues.  
+It can be a comma separated list of labels (e.g: `question,bug`).
+
+If unset (or an empty string), this option will not alter the stale workflow.
+
+Default value: unset
+
+#### exempt-pr-labels
+
+The label(s) that can exempt to automatically mark as stale the pull requests.  
+It can be a comma separated list of labels (e.g: `need-help,WIP`).
+
+If unset (or an empty string), this option will not alter the stale workflow.
+
+Default value: unset
+
+#### only-labels
+
+A white-list of label(s) to only process the issues or the pull requests that contain all these label(s).  
```

Comment:
```suggestion
A allow-list of label(s) to only process the issues or the pull requests that contain all these label(s).  
```

I think the trend is to use allow-list and block-list instead of white-list and black-list.

Reply:
```suggestion
An allow-list of label(s) to only process the issues or the pull requests that contain all these label(s).  
```